src/modules/colors/ColorView.js

```diff
@@ -1,24 +1,39 @@
 import React, {PropTypes, Component} from 'react';
 import {
-  Text,
+  Button,
   View,
   StyleSheet
 } from 'react-native';
 
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
 const color = () => Math.floor(255 * Math.random());
 
 /**
- * Sample view to demonstrate navigation patterns.
+ * Sample view to demonstrate StackNavigator
  * @TODO remove this module in a live application.
  */
 class ColorView extends Component {
   static displayName = 'ColorView';
 
+  static navigationOptions = {
+    title: 'Colors!',
+    tabBar: () => ({
+      icon: (props) => (
+        <Icon name='color-lens' size={24} color={props.tintColor} />
+      )
+    }),
+    // TODO: move this into global config?
```

Comment:
Do you still want to move the header config somewhere else? Otherwise let's remove the TODO ;)